Include/Library/OcCpuLib.h

```diff
@@ -67,6 +67,7 @@ typedef struct {
   UINT16                  PackageCount;
   UINT16                  CoreCount;
   UINT16                  ThreadCount;
+  BOOLEAN                   Hypervisor;
```

Comment:
That seems to have an extra space in the declaration, and I believe we can move it above to reduce the structure size. For example, before `Type`.

Reply:
I'm not quite understand your point and not quite familar with those structuration.Can you give a more specific example?